share/spice/yacht_specs/content.handlebars

```diff
@@ -0,0 +1,32 @@
+<!-- Div for the yacht specifications -->
+<div class='thumb'>
+    
+    <div class='image'>
```

Comment:
Whoops, the function is meant to return an object. I'll update the snippet above